app/editor/package.json

```diff
@@ -56,7 +56,7 @@
     "react-tooltip": "5.10.0",
     "redux-logger": "3.0.6",
     "styled-components": "5.3.9",
-    "tno-core": "0.0.276"
+    "tno-core": "0.0.279"
```

Comment:
Used the tno-core version from Jonny's PR since it fixes the missing `aliases` field on `IMinisterModel`